nilearn/plotting/displays.py

```diff
@@ -206,8 +206,9 @@ class GlassBrainAxes(BaseAxes):
     volumes with a schematic view of the brain.
 
     """
-    def __init__(self, ax, direction, coord, **kwargs):
+    def __init__(self, ax, direction, coord, plot_negative=False, **kwargs):
```

Comment:
> > Maybe plot_absolute=True?

> signed_colormap?

Thinking about it, I prefer 'plot_absolute', or maybe 'plot_abs', because
it will be obvious the link to np.abs.